### PR TITLE
Fix(stylekit): Styling adjustments

### PR DESCRIPTION
**Pull request by ghost**
Adjust styles on multiple elements like charts, input and buttons for better render.

### CODE REVIEW

File: gui/src/components/Taipy/Input.tsx
Comment (FredLL-Avaiga) [inline gui/src/components/Taipy/Input.tsx]: You're modifying the default behavior here
margin dense will only be applied when stylekit is used ...

**Review by FredLL-Avaiga — COMMENTED** [review]
Wondering about input margin dense ...

File: gui/src/components/Taipy/Input.tsx
Comment (ghost) [reply gui/src/components/Taipy/Input.tsx]: Yup, I just took conscience of that and it's already fixed in my last commit.

File: gui/src/themes/stylekit.ts
Comment (FredLL-Avaiga) [inline gui/src/themes/stylekit.ts]: Should we then remove this ?

Comment (ghost) [reply gui/src/themes/stylekit.ts]: I was wondering whether or not to leave it to cover potential use cases that I may not be aware of, but if you think I have every FormControl component covered with margin="dense" in those 3 components, I can remove it, yes.

Comment (ghost) [reply gui/src/themes/stylekit.ts]: Done

File: gui/src/components/Taipy/DateSelector.tsx
Comment (FredLL-Avaiga) [inline gui/src/components/Taipy/DateSelector.tsx]: This will be removed with the Date Selector PR
Can you remove the modification on this file for ease of merge ?

File: gui/src/components/Taipy/Input.tsx
Comment (FredLL-Avaiga) [inline gui/src/components/Taipy/Input.tsx]: No need to modify an unchanged file

File: gui/src/components/Taipy/DateSelector.tsx
Comment (ghost) [reply gui/src/components/Taipy/DateSelector.tsx]: Done

Comment (ghost) [reply gui/src/components/Taipy/DateSelector.tsx]: Done

File: gui/src/components/Taipy/Input.tsx
Comment (ghost) [reply gui/src/components/Taipy/Input.tsx]: Done

**Review by FredLL-Avaiga — APPROVED** [review]

**Comment by ghost** [comment]
@FredLL-Avaiga if this looks nice to @FabienLelaquais as well, can I let one of you merge this, please ? I'll be qui busy after this morning.
In the worst case, I'll deal with it on Monday.
Thank you! 🙂

**Review by FabienLelaquais — APPROVED** [review]
👍

**Pull request merged**
